File: packages/android/src/main/kotlin/gov/civui/components/CivToolbar.kt
// CivUI — CivToolbar for Jetpack Compose
// Above-grid toolbar container. Mirrors civ-toolbar.
//
// Placeholder body — see .claude/rules/audit-debt.md → "Native platform implementation pass".

package gov.civui.components

import androidx.compose.foundation.layout.Row
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

@Composable
fun CivToolbar(
    label: String = "",
    modifier: Modifier = Modifier,
    content: @Composable () -> Unit = {},
) {
    Row(modifier = modifier) {
        // Placeholder — see audit-debt.md for native implementation status.
        content()
    }
}

File: packages/ios/Sources/CivUI/CivToolbar.swift
// CivUI — CivToolbar for SwiftUI
// Above-grid toolbar container. Mirrors civ-toolbar.
//
// Placeholder body — the prop surface satisfies schema parity. See
// .claude/rules/audit-debt.md → "Native platform implementation pass".

import SwiftUI

public struct CivToolbar: View {
    /// Accessible label for the toolbar role.
    public var label: String

    public init(label: String = "") {
        self.label = label
    }

    public var body: some View {
        EmptyView()
    }
}
